RecoTracker/CkfPattern/plugins/GroupedCkfTrajectoryBuilder.cc

```diff
@@ -836,15 +835,13 @@ void GroupedCkfTrajectoryBuilder::rebuildSeedingRegion(const TrajectorySeed& see
   auto hitCloner = static_cast<TkTransientTrackingRecHitBuilder const*>(hitBuilder())->cloner();
   KFTrajectoryFitter fitter(backwardPropagator(seed), &updator(), &estimator(), 3, nullptr, &hitCloner);
   //
-  TrajectorySeed::range rseedHits = seed.recHits();
   std::vector<const TrackingRecHit*> seedHits;
   //seedHits.insert(seedHits.end(), rseedHits.first, rseedHits.second);
```

Comment:
These commented out lines become now obsolete: better to remove.
If you really want to keep them, then please resurrect (commented out) also 
```C++
  TrajectorySeed::range rseedHits = seed.recHits();
```

Reply:
I think the commented out code can be removed. Just a loop that fills a vector from each element of the original one is not very interesting to keep IMO